src/explicitode.jl

```diff
@@ -373,7 +369,7 @@ function taylorinteg{T<:Number}(f, q0::Array{T,1}, t0::T, tmax::T,
         end
     end
 
-    return view(tv,1:nsteps), view(xv,:,1:nsteps)' #for xv, first do view, then transpose (otherwise it crashes)
+    return view(tv,1:nsteps), view(transpose(xv),1:nsteps,:)
```

Comment:
Just tried both versions and find no difference (first `transpose` then `view`, and viceversa), I think that upon calling `transpose`, a new array is allocated, and it doesn't matter if we transpose either before of after doing `view`.. So if we can choose, perhaps the code is more understandable writing `view(transpose(xv),1:nsteps,:)`, instead of `view(xv,:,1:nsteps)'`